app/assets/javascripts/discourse/controllers/convert-to-public-topic.js.es6

```diff
@@ -0,0 +1,26 @@
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Ember.Controller.extend({
+  publicCategoryId: null,
+  saving: true,
+
+  onShow() {
+    this.publicCategoryId = null;
+    this.saving = false;
+  },
+
+  actions: {
+    makePublic() {
+      let topic = this.model;
```

Comment:
I would say this variable here is not very useful, maybe to make it explicit it's a topic?

Reply:
Yes, and the variable is used 3 times, so I felt it was slightly better than `this.model` three times.